mmv1/templates/terraform/pre_delete/compute_region_network_endpoint.go.tmpl

```diff
@@ -27,7 +27,6 @@ if fqdnProp != "" {
   toDelete["fqdn"] = fqdnProp
 }
 
-{{- if ne $.TargetVersionName "ga" }}
```

Comment:
Nit: for this file and possibly the test file, if they no longer need beta-only behavior, you could change them to just be `.go` files. Of course, if you will just be adding more beta fields again soon, then it might be easier just to keep them this way.